homeassistant/components/stream/services.yaml

```diff
@@ -1,6 +1,9 @@
 record:
-  description: Make a .mp4 recording from a provided stream.
+  description: Make a .mp4 recording from a provided stream. This is an internal service meant to be called by other components.
```

Comment:
Internal components can call it directly on the `Stream` object that they received from calling the internal cross-integration API `create_stream`. I think that we can drop this service.

Reply:
Ok great, I was proposing above we add a method called record similar to the method that gives back the url.  I'll try this out and see how it looks.